ui/src/app/base/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,101 @@
+import { useEffect } from "react";
+
+import type { UsabillaLive } from "app/base/types";
+
+declare global {
+  export interface Window {
+    usabilla_live: UsabillaLive;
+    lightningjs: {
+      require: (variable: string, url: string) => Window["usabilla_live"];
+    };
+  }
+}
+
+type Props = {
+  debug?: boolean;
+  enableAnalytics?: boolean | null;
+  version: string;
+};
+
+export const Footer = ({
```

Comment:
I've copied these components over and not removed the ones in shared as they're still required by legacy.